core/src/main/java/io/questdb/cairo/StringTypeDriver.java

```diff
@@ -30,12 +30,17 @@
 import io.questdb.std.Unsafe;
 import io.questdb.std.Vect;
 import io.questdb.std.str.LPSZ;
+import org.jetbrains.annotations.Nullable;
 
 import static io.questdb.cairo.ColumnType.LEGACY_VAR_SIZE_AUX_SHL;
 
 public class StringTypeDriver implements ColumnTypeDriver {
     public static final StringTypeDriver INSTANCE = new StringTypeDriver();
 
+    public static long getPlainValueByteCount(@Nullable CharSequence value) {
```

Comment:
`long` return values is confusing and inconsistent. It should be `int`

Reply:
Changed `long` to `int` in `VarcharTypeDriver` in 8ef6c94e54bd442f3a6e30190ca6c5623de756b1

For a string of `Integer.MAX_VALUE` length, the bytes count will be 4294967298 which doesn't fit into an `int`. So, we should probably keep `long` type for the `StringTypeDriver`. WDYT?